SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -493,10 +500,35 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 };
 
 template <class T>
-T* ASpatialFunctionalTest::SpawnActor(const FActorSpawnParameters& SpawnParameters, const bool bRegisterAsAutoDestroy)
+T* ASpatialFunctionalTest::SpawnActor(const bool bRegisterAsAutoDestroy)
+{
+	T* Actor = GetWorld()->SpawnActor<T>();
+	verifyf(IsValid(Actor), TEXT("Actor returned by GetWorld->SpawnActor must be valid."));
+
+	if (bRegisterAsAutoDestroy)
+	{
+		RegisterAutoDestroyActor(Actor);
+	}
+	return Actor;
```

Comment:
I prefer just calling the most "powerful" function directly.

Reply:
+1

Reply:
changed!